packages/docker-engine/docker-engine.spec

```diff
@@ -32,6 +32,8 @@ Source1000: clarify.toml
 # Bottlerocket-specific - Privileged containers should receive SELinux labels
 # https://github.com/bottlerocket-os/bottlerocket/issues/1011
 Patch0001: 0001-bottlerocket-privileged-shouldn-t-disable-SELinux.patch
+# Update aws-sdk-go for IMDSv2 support
+Patch0100: awslogs_update_aws-sdk-go_to_support_imdsv2.patch.bz2
```

Comment:
Why did you skip to 100 instead of going to 2?

Reply:
Tom's suggestion. To distinguish between two patches to different package.

Reply:
I assume 1-N will be a series of patches you (@samuelkarp) write for Bottlerocket-specific changes, and generally patch series from different sources will be in different numbered chunks, like jumping up to 100 or 1,000.